templates/cf-hub.service.mustache

```diff
@@ -2,12 +2,16 @@
 Description=CFEngine Enterprise Hub Report Collector
 PartOf=cfengine3.service
 ConditionPathExists={{{vars.sys.bindir}}}/cf-hub
+ConditionPathExists={{{vars.sys.workdir}}}/inputs/promises.cf
 After=syslog.target
 After=network.target
 
+Wants=cf-postgres.service
+After=cf-postgres.service
```

Comment:
Not requires?
I guess this is a sycing task, so even if we want to, thats a different task.

Reply:
Well. I am syncing based on what's in core/misc/systemd so any qualms we should address in both places. :+1: I was just working on this Wants versus Requires recently due to maybe problems with HA but @vpodzime mentioned that as-is things seem to work fine.